### PR TITLE
readme update for fluentd-gcp-scaler

### PR DESCRIPTION
**Pull request by x13n**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

/assign @crassirostris

### CODE REVIEW

File: cluster/addons/fluentd-gcp/README.md [hunk 1]
```diff
@@ -8,4 +8,67 @@ they can be searched, viewed, and analyzed.
 
 Learn more at: https://kubernetes.io/docs/tasks/debug-application-cluster/logging-stackdriver
 
+## Troubleshooting
+
+In Kubernetes clusters in version 1.10.0 or later, fluentd-gcp DaemonSet can be
+manually scaled. This is useful e.g. when the applications running in the
```
Comment (crassirostris) [inline cluster/addons/fluentd-gcp/README.md]: `...when applications...`

File: cluster/addons/fluentd-gcp/README.md [hunk 2]
```diff
@@ -8,4 +8,67 @@ they can be searched, viewed, and analyzed.
...
+cluster are sending a large volume of logs (i.e. over 100kB/s), causing
+fluentd-gcp to fail with OOM errors. Conversely, if the applications aren't
```
Comment (crassirostris) [inline cluster/addons/fluentd-gcp/README.md]: `...with OutOfMemory errors.`

File: cluster/addons/fluentd-gcp/README.md [hunk 3]
```diff
@@ -8,4 +8,67 @@ they can be searched, viewed, and analyzed.
...
+generating a lot of logs, it may be useful to reduce the amount of resources
+consumed by fluentd-gcp, making them available to other applications. To learn
```
Comment (crassirostris) [inline cluster/addons/fluentd-gcp/README.md]: `...making these resources available...`

File: cluster/addons/fluentd-gcp/README.md [hunk 4]
```diff
@@ -8,4 +8,67 @@ they can be searched, viewed, and analyzed.
...
+more about Kubernetes resource requests and limits, see the official
+documentation ([CPU][cpu], [memory][memory]). The amount of resources requested
+by fluentd-gcp on every node in the cluster can be fetched by running following
+command:
+
+```
+$ kubectl get ds -n kube-system -l k8s-app=fluentd-gcp -o custom-columns=NAME:.metadata.name,CPU_REQUEST:.spec.template.spec.containers[].resources.requests.cpu,MEMORY_REQUEST:.spec.template.spec.containers[].resources.requests.memory,MEMORY_LIMIT:.spec.template.spec.containers[].resources.limits.memory
```
Comment (crassirostris) [inline cluster/addons/fluentd-gcp/README.md]: Make this command multiline maybe?

**Review by crassirostris — COMMENTED** [review]
LGTM

Several nits

**Comment by x13n** [comment]
Thanks! Updated.

**Pull request merged**